### PR TITLE
gamepage slot machine styling

### LINKED ISSUE #81: Gamepage slot machine styling
**Is your feature request related to a problem? Please describe.**
Replacing the png slot machine and turning it into an actual element for more flexibility of styling.

### PR DESCRIPTION
**Pull request by gonzalespaulb**
## Changes
1. Slot machine styling
2. Slot machine animation functionality

## Purpose
The main animation for when the pokemon randomizer function is running in the background

## Approach
SCSS and Keyframes. Dynamically adds active class when the reel states are true

Closes #81

### CODE REVIEW

File: pokeapp/src/components/GamePage.js [hunk 1]
```diff
@@ -26,13 +34,142 @@ const GamePage = ({makeUpperCase, setIsPokeList}) => {
...
+      const roller1 = () => {
+
+        let reelStrip = [];
+        if (reel1) {
+          for (let current = 1; current <= NUMBER_OF_REEL_FACES; current++ ) {
+            reelStrip.push(
+              <>
+              <div style={rollerStyle} className="roller-img roller-ani"/>
+              <div style={rollerStyle2} className="roller-img roller-ani"/>
+              </>
```
Comment (kmausolf) [inline pokeapp/src/components/GamePage.js:56]: We might need to key this block.  Are you getting console warnings about it?

File: pokeapp/src/assets/pokemons/index.js [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+// -------------------------------------------------------------------------------
+
+export const slotMachinePokemon = (pokemon) => {
+    switch (pokemon) {
+        case `aerodactyl`:
+            return aerodactyl;
+        case `articuno`:
+            return articuno;
+        case `bellsprout`:
```
Comment (Gjhnsn) [inline pokeapp/src/assets/pokemons/index.js:77]: Slot machine pokemon 👀
cant wait to see this!

**Review by kmausolf — COMMENTED** [review]
This is really cool to see in action.  Nice work on solving this!

File: pokeapp/src/styling/components/_gamepage.scss [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+.lever-stick-active  {
+    animation: 1s ease 0s 1 normal forwards running lever-stick-down;  
+}
+
+@keyframes lever-stick-down {
+    0% { transform: translateY(-90%) scaleY(1); }
+    50% { transform: translateY(-40%) scaleY(0.5); }
+    100% { transform: translateY(-90%) scaleY(1); }
+}
```
Comment (Gjhnsn) [inline pokeapp/src/styling/components/_gamepage.scss:156]: Excited to see all this in action, looks like a lot of movement happening!

**Review by Gjhnsn — APPROVED** [review]
Great work! Cant wait to see all the animations

File: pokeapp/src/components/GamePage.js (hunk 1 above)
Comment (gonzalespaulb) [reply pokeapp/src/components/GamePage.js:56]: Definitely console warnings about a key prop. This is getting pushed into the reel strip from a loop so would it be ok to use the "current" value in the for loop (line 51 for example)?

File: pokeapp/src/assets/pokemons/index.js [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+        case `weezing`:
+            return weezing;
+        case `wobbuffet`:
+            return wobbuffet;
+        case `zangoose`:
+            return zangoose;
+        case `zapdos`:
+            return zapdos;
+    }
+}
```
Comment (BlackCubes) [inline pokeapp/src/assets/pokemons/index.js:174]: WAH!!!!!!!!!!! So excited

File: pokeapp/src/components/GamePage.js [hunk 5]
```diff
@@ -1,10 +1,18 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Navigation from './Navigation';
 import PokemonCard from './PokemonCard';
 import { useSelector } from 'react-redux';
+import {slotMachinePokemon} from '../assets/pokemons/index';
+import pikachu from '../assets/rollers/pikachu-roller.png';
+import pokeball from '../assets/rollers/pokeball-roller.png';
```
Comment (BlackCubes) [inline pokeapp/src/components/GamePage.js:7]: 👀

File: pokeapp/src/components/GamePage.js (hunk 1 above)
Comment (BlackCubes) [reply pokeapp/src/components/GamePage.js:56]: Maybe on line 53, put `<React.Fragment>` with a `key` inside to make this work without warnings????

**Review by BlackCubes — APPROVED** [review]
Looks so goood!!! Super excited!

File: pokeapp/src/components/GamePage.js (hunk 1 above)
Comment (gonzalespaulb) [reply pokeapp/src/components/GamePage.js]: @BlackCubes That was exactly the fix! Thank you.

**Pull request merged**